decidim-core/db/seeds.rb

```diff
@@ -13,6 +13,7 @@
     password_confirmation: "decidim123456",
     organization: staging_organization,
     confirmed_at: Time.current,
+    locale: "en",
```

Comment:
Could you use `I18n.default_locale` instead? This way it will work in a third-party application that changes the default locale.